### PR TITLE
628: Add missing application fields

### LINKED ISSUE #628: [Application] Add missing options for applications
Currently, only a small subset of the application form is implemented.
Add fields for the remaining possible requirements that can entitle an applicant to apply for an EAK.
Here's a list of all possible requirements:
https://www.lbe.bayern.de/engagement-anerkennen/ehrenamtskarte/voraussetzungen/index.php

### PR DESCRIPTION
**Pull request by michael-markl**
I just realized that I still had this branch locally.
It's too large (sorry for that), but it closes #628 .

Let me know if I should try to break it up into several PRs.

### CODE REVIEW

File: administration/src/application/components/forms/OrganizationForm.tsx [hunk 1]
```diff
@@ -21,13 +22,19 @@ const organizationCategoryOptions = {
   ],
 }
 
+const contactHasGivenPermissionOptions = {
+  required: true,
+  notCheckedErrorMessage: 'Die Kontaktperson muss zugestimmt haben, damit Sie Ihren Antrag senden können.',
+} as const
```
Comment (maxammann) [inline administration/src/application/components/forms/OrganizationForm.tsx:28]: TS surprises me :D Why "as const" (No need to change I'm just curious)

File: administration/src/application/components/forms/VolunteerServiceEntitlementForm.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      <CustomDivider label='Angaben zur Tätigkeit' />
+      <ShortTextForm.Component
+        label='Name des Programms'
+        state={state.programName}
+        setState={useUpdateStateCallback(setState, 'programName')}
+      />
+      <h4>Tätigkeitsnachweis</h4>
+      <p>
+        Hängen Sie hier bitte einen eingescannten oder abfotografierten Tätigkeitsnachweis an. Die Datei darf maximal{' '}
+        {FILE_SIZE_LIMIT_MEGA_BYTES} MB groß sein und muss im JPG, PNG oder PDF Format sein.
```
Comment (maxammann) [inline administration/src/application/components/forms/VolunteerServiceEntitlementForm.tsx:52]: Minor: We could create a separate Component for this sentence.

File: administration/src/index.tsx [hunk 3]
```diff
@@ -11,7 +11,11 @@ import App from './App'
 import reportWebVitals from './reportWebVitals'
 
 const root = createRoot(document.getElementById('root')!)
-root.render(<App />)
+root.render(
+  <StrictMode>
```
Comment (maxammann) [inline administration/src/index.tsx:15]: Warning: This might break other PRs which do not merge in whitelabel first.

**Review by maxammann — APPROVED** [review]
That looks like a crazy amount of work! Good job! 🎉

File: administration/src/index.tsx (hunk 3 above)
Comment (michael-markl) [reply administration/src/index.tsx:15]: I think the probability is very low. If it's the case, then we'll have to fix it afterwards.

File: administration/src/application/components/forms/VolunteerServiceEntitlementForm.tsx (hunk 2 above)
Comment (michael-markl) [reply administration/src/application/components/forms/VolunteerServiceEntitlementForm.tsx:52]: I agree...

**Comment by michael-markl** [comment]
Would be cool, if someone could smoke test this PR a bit.

**Comment by maxammann** [comment]
Smoke tested, works!

I created an application with the new fields and checked it in the backend.

rdy for merge

**Pull request merged**